changelog.txt

```diff
@@ -1,4 +1,6 @@
 V1.XX.X
+[QMS-158] Change Routino search for [prefix-]profiles.xml.
+Searches routino database path (user defined and writable) and default global location (not user writable). Allows user defined profiles.
```

Comment:
As I already mentioned:
* Adjust the title of the ticket if it's not really summarizing the PR
* Add theticket number and the  title to the change log
* Keep the order of the issues.

Just have a look at the other entries and you get an idea of how to do it right.